galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/internal/HttpClientImpl.java

```diff
@@ -508,21 +507,24 @@ public IHttpClient setAuthorisation(String username, String password, URI scope)
      */
     public IHttpClient build() {
 
-        HttpClientBuilder builder = HttpClients.custom().setDefaultCookieStore(cookieStore);
+    	RequestConfig.Builder requestBuilder = RequestConfig.custom();
+    	HttpClientBuilder builder = HttpClientBuilder.create();
+    	builder.setDefaultCookieStore(cookieStore);
+    	requestBuilder.setCookieSpec(CookieSpecs.STANDARD);
         builder.setDefaultCredentialsProvider(credentialsProvider);
         builder.setDefaultHeaders(commonHeaders);
 
         if (timeout > 0) {
-            RequestConfig.Builder requestBuilder = RequestConfig.custom().setConnectTimeout(timeout)
-                    .setConnectionRequestTimeout(timeout).setSocketTimeout(timeout);
+            requestBuilder.setConnectTimeout(timeout)
+                          .setConnectionRequestTimeout(timeout).setSocketTimeout(timeout);
             builder.setDefaultRequestConfig(requestBuilder.build());
```

Comment:
this is being done on line 527,  so doesn't need to be done here.

Reply:
thanks for pointing that out, have removed this and kept on line 527